tests/test_params.py

```diff
@@ -598,11 +720,9 @@ class ParamSpider(Args[Params], Spider):
     }
 
 
+@pytest.mark.skipif(USING_PYDANTIC_1, reason="Requires Pydantic 2.x")
 def test_subclass_config_extension():
-    try:
-        from pydantic import ConfigDict
-    except ImportError:
-        pytest.skip("No pydantic.ConfigDict")
```

Comment:
1.10 has `ConfigDict` but doesn't have `BaseModel.model_config` so basically this doesn't work on all 1.x anyway.